includes/class-object-cache.php

```
@@ -115,42 +122,162 @@ public function get_status() {
+				$nodes       = $config['nodes'] ?? array();
+				$master_name = $config['master_name'] ?? 'mymaster';
+
+				if ( is_string( $nodes ) ) {
+					$nodes = array_filter( array_map( 'trim', explode( "\n", $nodes ) ) );
+				}
+
+				$errors = array();
+				foreach ( $nodes as $node ) {
+					list( $s_host, $s_port ) = array_pad( explode( ':', $node ), 2, 26379 );
+					try {
+						$sentinel = new \RedisSentinel(
```

PHPStan (2.1.46)

[error] Class RedisSentinel constructor invoked with 1 parameter, 2-6 required.


(arguments.count)